doc/examples/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@
 if (NGRAPH_CPU_ENABLE)
```

Comment:
Any reason why we only build examples if cpu backend is enabled? Interpreter too slow for mnist_mlp?

Reply:
Maybe it was to make sure they built once.